### DISCUSSION TITLE
Um artigo de introdução ao que é JAMStack

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#2117] [Dicas]
Galera, um artigo legal sobre JAMStack para quem ainda não está por dentro deste novo modelo de desenvolvimento que traz alguns prós como eficiencia, performance e custo e alguns contras também... Segue o link para quem achar interessante:

https://www.linkedin.com/pulse/voc%C3%AA-sabe-o-que-%C3%A9-jamstack-wilson-neto-b-r-gomes/

**Comment by nazarepiedady** [comment]
Para quem quiser tocar em algo um pouco mais abrangente: https://www.netlify.com/jamstack/
E se preferir um ebook: https://www.netlify.com/oreilly-jamstack/
